ci-operator/config/openshift/microshift/openshift-microshift-release-4.19.yaml

```diff
@@ -85,31 +85,6 @@ tests:
     env:
       DRY_RUN: "y"
     workflow: openshift-microshift-rebase
-- as: rebase-on-nightlies
-  cron: 0 4 * * 1-5
-  steps:
-    workflow: openshift-microshift-rebase
-- as: e2e-aws-tests-cache-nightly
-  capabilities:
-  - intranet
-  cron: 30 0 * * *
-  restrict_network_access: false
-  steps:
-    cluster_profile: aws-2
-    env:
-      MICROSHIFT_OS: rhel-9.6
-    workflow: openshift-microshift-e2e-metal-cache
-- as: e2e-aws-tests-cache-nightly-arm
-  capabilities:
-  - intranet
-  cron: 30 0 * * *
-  restrict_network_access: false
-  steps:
-    cluster_profile: aws-2
-    env:
-      EC2_INSTANCE_TYPE: c7g.2xlarge
-      MICROSHIFT_OS: rhel-9.6
-    workflow: openshift-microshift-e2e-metal-cache
```

Comment:
I'm not sure we should cancel cache builds. We still do this for 4.18 and previous releases.
The problem is that bootc builds need ostree images for some tests, and those can only come from cache.